docs/_basic/authenticating_oauth.md

```diff
@@ -15,7 +15,45 @@ You will need to provide your:
 ---
 
 ##### Installing your App
-Bolt for JavaScript provides an **Install Path** `/slack/install` out-of-the-box. This returns a simple `Add to Slack` button where users can initiate direct installs of your app. 
+Bolt for JavaScript provides an **Install Path** `/slack/install` out-of-the-box. This endpoint returns a simple `Add to Slack` button where users can initiate direct installs of your app. For example, if your app was hosted at `www.example.com`, you would be able to install your app at `www.example.com/slack/install`. 
```

Comment:
How about adding these?
```suggestion
Bolt for JavaScript provides an **Install Path** `/slack/install` out-of-the-box. This endpoint returns a simple `Add to Slack` button where users can initiate direct installs of your app with a valid `state` parameter. For example, if your app was hosted at `www.example.com`, you would be able to install your app at `www.example.com/slack/install`. If you would like to skip rendering the simple webpage and directly navigate end-users to Slack authorize URL, your app can set `installerOptions.directInstall: true` in the `App` constructor.

```

Reply:
@seratch This looks good! I just wanted to test this out, for some reason, when I try it in my app.js 

const app = new App({
  logLevel: LogLevel.DEBUG,
  signingSecret: process.env.SLACK_SIGNING_SECRET,
  clientId: process.env.SLACK_CLIENT_ID,
  clientSecret: process.env.SLACK_CLIENT_SECRET,
  stateSecret: 'my-state-secret',
  installerOptions: {directInstall: true},
  ...
  ```
  I would expect this to not render the page with the `add to slack` button but for some reason it still does. Is this expected?

Reply:
@horeaporutiu hmm, do you have `receiver` in the constructor arguments? If so, the option should be passed to `receiver` instead. Otherwise, this should work.

Reply:
@seratch Thank you. I found my mistake. I was overwriting the value further down in the constructor.